### DISCUSSION TITLE
Enhancement: You should make Friday Night Funkin for Nintendo Switch and Playstation 4 and 5 and plus every 2 or 3 fridays there should be a new Week update

### DISCUSSION THREAD

**Discussion by ThatOneGuyNamedKahron** [FunkinCrew/Funkin#1581] [Ideas]
# Please check for duplicates or similar issues before creating this issue.

**Comment by regulad** [comment]
IIRC there was a switch pitch a while ago but I doubt it's going to go through since the game is open source. Also there has been a new level every couple of weeks.

**Reply by KitOdin** [reply]
"Also there has been a new level every couple of weeks."
dude, we have been waiting for more than HALF OF THE GAMES LIFE SPAN waiting for a new update.

**Comment by AyderProgramic** [comment]
If you are going to port the game to Nintendo switch and Playstation 5, then to Xbox one, X |  S port too!

**Comment by Windows1Door2Daniel** [comment]
its expensive upload a game in worldwide consoles

PSN: $5000
NINTENDO: possibly free
XBOX: same to nintendo

well ps is the only expensive

**Comment by ghost** [comment]
> I will explain this:
> 
> It is possible building FNF for X?
> 
> 
> 
> - PlayStation: **no**.
> 
> - Nintendo Switch: **yes!**
> 
> - Nintendo Wii U: **it was possible, but for legal reasons, it isn't now**.
> 
> - Xbox One: **same with wii u**.
> 
> 
> 
> About PS5 and Xbox Series X & S, there isn't any way to build lol.

im confused, if the nintendo switch is allowed for you to build for, then why not the wii u?

**Comment by Windows1Door2Daniel** [comment]
i saw mufin tried to put fnf to switch

but got rejected by NinJa

**Reply by libauror-a** [reply]
so he rejected his own idea

**Reply by Windows1Door2Daniel** [reply]
I am sure i was meaning about Nintendo but i wrote this a time ago but he didn't

Nintendo did

Ninja wanted the game to be in the Indie Nintendo program but it got rejected or thats what i think
